### DISCUSSION TITLE
ARP Timeout

### DISCUSSION THREAD

**Discussion by Raizel-sys** [netalertx/NetAlertX#907] [Q&A]
Hello,

I have a malfunction.
I have installed the tool but all the discoveries fall into timeout.
However, when I run the ARP command on the machine directly, the machine retrieves the assets on the scanned network.
Any ideas?
![2024-12-05 09_24_42-NetAlertX - srv-demo-lnx-netx – Brave](https://github.com/user-attachments/assets/f2274205-5605-45bf-aebe-e718a964d0a6)
![2024-12-05 09_28_49-NetAlertX - srv-demo-lnx-netx – Brave](https://github.com/user-attachments/assets/75dcdae3-543a-4c32-83b4-5fb8db8c9e9d)

**Comment by jokob-sk** [comment]
Hey,

Did you try to increase the default timeout?

Also the /26 subnet seems odd... Is taht allowed?

**Comment by jokob-sk** [comment]
Please make sure you use the right network mask - if unsure check About -> System Info -> Hardware Info. Use that mask to run arpscan in the container to make sure your settings are correct. Please double check these docs to see how to test the arpscan command: https://github.com/jokob-sk/NetAlertX/blob/main/docs/SUBNETS.md

**Comment by ingoratsdorf** [comment]
I don't see a timeout but I clearly see a [NMAPDEV] ERROR: CalledProcessError.
Meaning nmap cannot process the arguments, ie your rather strange subnet mask.
Should probably be /16 or /24.

Note: nmap on a *.*.*.0/16 subnet will take AGES. Scanning 65536 hosts.... at a second or so timing. Good luck.
